### PR TITLE
Open source Card

### PR DESCRIPTION
**Pull request by ujju20**
Issue #877 
I have added roughly open source cards just wanted to know that am I going right?
Also, I am unable to show images even I am giving the right path.

### CODE REVIEW

File: src/pages/Experience/Experience.js [hunk 1]
```diff
@@ -1,9 +1,146 @@
 import React from 'react';
+import OpenSourceCard from './open-source-card';
 
 const Experience = () => {
+    const openSourcePrograms = [
+        {
+          title: "GirlScript Summer Of Code 2020",
```
Comment (smaranjitghose) [inline src/pages/Experience/Experience.js:7]: [Sample Reference for using assets like images](https://github.com/smaranjitghose/doc2pen/blob/master/src/pages/Home/sections/OpenSource/OpenSource.jsx)

File: src/pages/Experience/Experience.js [hunk 2]
```diff
@@ -1,9 +1,146 @@
...
+          cardImage: "src/assets/images/open-sorce/1.jpg",
+          description:
+            "Responsible for handling the projects GirlScript App and GirlScript Website Boilerplate.",
+        },
+        {
+          title: "StudentCode-in 2020",
+          cardImage: "src/assets/images/open-source/2.jpg",
```
Comment (smaranjitghose) [inline src/pages/Experience/Experience.js:14]: Have a look at the above reference for the structure of pages and the corresponding sections.

Restructure this into the open-source section. 

Try to use JSX

File: src/pages/Experience/Experience.js [hunk 3]
```diff
@@ -1,9 +1,146 @@
...
+                    </g>
+                    <path id="Vector_104" opacity="0.1" d="M128.133 248.132C128.133 248.132 130.717 247.335 131.79 248.847L128.133 248.132Z" fill="black"></path>
+                    <path id="Vector_105" opacity="0.1" d="M188.013 248.448C188.013 248.448 183.732 248.198 183.9 249.356L188.013 248.448Z" fill="black"></path>
```
Comment (smaranjitghose) [inline src/pages/Experience/Experience.js:121]: Define this in a separate component called top_illus

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
@ujju20 Any updates?

**Comment by ujju20** [comment]
> @ujju20 Any updates?

Ya, I am doing it I will push changes today itself.

**Comment by ujju20** [comment]
I am Uable to render topIlus in the experience.js file I don't know why it is not getting rendered.

**Review by anushbhatia — COMMENTED** [review]
image
 it is rendered have a look in the deployment. The functionality may not work cause code has not been integrated

**Comment by smaranjitghose** [comment]
> I am Unable to render topIlus in the experience.js file I don't know why it is not getting rendered.

For now, that is not your responsibility. You have to make sure that when we visit the `/research` path, we can see the heading "Open Source" followed by the cards of the design as requested in the corresponding issue description. The cards should be mobile responsive, have real data instead of lorem ipsum, and be injected dynamically by updating the value of a variable

**Comment by smaranjitghose** [comment]
I have 1 more day for this

**Comment by ujju20** [comment]
> I have 1 more day for this
Ok I have made the changes 
Check them.

**Comment by ujju20** [comment]
Ok I have made the changes
Check them.

**Comment by anushbhatia** [comment]
Deploy is failing check if there are any warnings in your code remove them if any.

**Comment by ujju20** [comment]
There was one warning I have fixed that 
see this 
Screenshot from 2021-07-13 12-35-22

**Comment by ujju20** [comment]
Have You reviewed my PR?

**Comment by smaranjitghose** [comment]
Why is the old UI used for this? Even after having given detailed description in the corresponding issue.

**Pull request closed without merging**